impl/client/src/main/doc-js/pentaho/visual/base/spec/IView.jsdoc

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright 2010 - 2017 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The `IView` interface represents the set of options used to initialize a view instance.
+ *
+ * @name pentaho.visual.base.spec.IView
+ * @interface
+ * @extends pentaho.type.spec.IComplex
+ *
+ * @see pentaho.visual.base.View
+ */
+
+/**
+ * The width that the container application has allocated to display the view
+ * without horizontal or vertical scrolling, in pixels.
+ *
+ * This property is required.
+ *
+ * @name width
+ * @memberOf pentaho.visual.base.spec.IView#
+ * @type {number}
+ * @see pentaho.visual.base.View#width
+ */
+
+/**
+ * The height that the container application has allocated to display the view
+ * without horizontal or vertical scrolling, in pixels.
+ *
+ * This property is required.
+ *
+ * @name height
+ * @memberOf pentaho.visual.base.spec.IView#
+ * @type {number}
+ * @see pentaho.visual.base.View#height
+ */
+
+/**
+ * The usage intention of the view, namely if interaction is enabled or not.
+ *
+ * For example, a container application can specify a value of `false`
+ * when rendering a view for printing purposes.
+ *
+ * @name isInteractive
```

Comment:
I would then propose to simply remove the property for the beta and think of a better alternative for later. We can test if currently setting it to `true`, when printing, comes ith any actual harm.